### DISCUSSION TITLE
Дать возможность выбора способок в ресте

### DISCUSSION THREAD

**Discussion by Shootnik01** [Cosmic-Overlord/TerraGov-Marine-Corps#300] [Ideas]
Не использования, а именно выбора. Чтобы сменить способку, и, встав, юзнуть ее, а не сначала вставать, чтобы юзать. Переключение режимов/видов способок (типо тоггл савадж у руни) тоже входит сюда

**Comment by fe-ed** [comment]
Мне кажется придется переписывать какой-то супер древний дряхлый еле рабочий кусок кода, но звучит весьма удобно

**Reply by Blundir** [reply]
Это правда, там нужно код блока использования менять, очень грязный метод для модульности, а ещё будет огромное количество новых багов, так что лучше не стоит.
